plugin/pkg/admission/defaultpodnetworkreadiness/admission_test.go

```diff
@@ -84,7 +84,7 @@ func TestAdmit(t *testing.T) {
 				t.Fatalf("Unexpected error returned from admission handler: %v", err)
 			}
 
-			v, ok := pod.Annotations[networkReadiness]
+			v, ok := pod.Annotations["arktos.futurewei.com/network-readiness"]
```

Comment:
I think there is constant defined for ""arktos.futurewei.com/network-readiness"

Reply:
yes there is - but in [another PR](https://github.com/futurewei-cloud/arktos/pull/427) that is already under review.

Reply:
it is intentional; in test code, I prefer to using hand-craft string instead of simple reference to const definition.